pkg/cmd/roachtest/cluster.go

```diff
@@ -2187,6 +2187,29 @@ func (c *clusterImpl) StartE(
 			return err
 		}
 	}
+
+	if startOpts.WaitForReplicationFactor > 0 {
+		l.Printf("waiting for replication factor of at least %d", startOpts.WaitForReplicationFactor)
+		var nodes option.NodeListOption
+		for _, o := range opts {
```

Comment:
Nit: worth refactoring into something like `nodes := toNodes(opts)` since it's used verbatim in `MakeNodes`.

Reply:
Done.